RetrospectiveExtension.Frontend/components/feedbackItem.tsx

```diff
@@ -253,6 +256,13 @@ export default class FeedbackItem extends React.Component<IFeedbackItemProps, IF
     this.hideRemoveFeedbackItemFromGroupConfirmationDialog();
   }
 
+  private setDisabledFeedbackItemDeletion = async (boardId: string, id: string) => {
+    const feedbackItem = await itemDataService.getFeedbackItem(boardId, id);
+    if (feedbackItem) {
+      this.setState({isDeletionDisabled: feedbackItem.upvotes > 0});
```

Comment:
@polatengin Should we disable deleting items from the feedback board when the stage is moved into Group (or out of collection)? I think disabling deletion with >0 votes is a good first step but we should think about freezing the board right after feedback is collected

Reply:
it's a very good suggestion, here is my 2 cents;

during the session, fundamentally, there is no single point of moving into phases, someone can stay in the `Collect` phase, when others moved into `Grouping` phase, etc.

I'm not sure if we should disable some of the functionality to some people but let it for others, I don't feel comfortable about it.

I'm not against it, just not sure 😄

I always think of the offline method of retro sessions, like, before a wall, people are standing, sticking post-its, moving them around, etc. my experience with such sessions is, people are doing it asynchronously. some people might be still giving feedbacks when others started grouping.

that's why we're not syncing people on the same stage but let them do their thing 😄

I'd like to discuss it with a broader group, it seems it's more of a `usability` / `user experience` issue, then a technical issue, to me. we'd may be doing wrong thing if we try to solve this from a technical point of view.

thoughts?

Reply:
I've created a work item for us to discuss more significant changes, if necessary: https://dev.azure.com/csebostoncrew/Retrospectives-AzDO-Extension/_sprints/taskboard/Retrospectives-AzDO-Extension%20Team/Retrospectives-AzDO-Extension/Sprint%202?workitem=1487

Reply:
thanks, since we have a work item to track, you can resolve this, I think.